applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -110,13 +110,22 @@ static void stream_sent_cb(struct bt_audio_stream *stream)
 static void stream_started_cb(struct bt_audio_stream *stream)
 {
 	int ret;
+	bool all_streams_streaming = true;
 
-	memset(seq_num, 0, ARRAY_SIZE(seq_num));
+	LOG_INF("Broadcast source %p started", (void *)stream);
 
-	ret = ctrl_events_le_audio_event_send(LE_AUDIO_EVT_STREAMING);
-	ERR_CHK(ret);
+	for (size_t i = 0U; i < ARRAY_SIZE(streams); i++) {
+		if (streams[i].ep->status.state != BT_AUDIO_EP_STATE_STREAMING) {
+			all_streams_streaming = false;
+		}
+	}
 
-	LOG_INF("Broadcast source %p started", (void *)stream);
+	if (all_streams_streaming) {
+		memset(seq_num, 0, ARRAY_SIZE(seq_num));
+
+		ret = ctrl_events_le_audio_event_send(LE_AUDIO_EVT_STREAMING);
+		ERR_CHK(ret);
+	}
```

Comment:
Since `seq_num` is an array, why not just set it to 0 for each stream where this callback is called instead?

Reply:
Hmm, to my understanding we want to clear all seq_num at the same time?
We want the left and right channel package with the "corresponding audio data" to have the same seq_num to tell the controller that these two packages should be sent in the same interval.

Reply:
This callback is currently just being called in a for loop in the host, so it's quite likely that it would work just fine. Alternatively, you could/should just keep a single `seg_num` for the BIG, instead of an array.